.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+**A Helpful Guide For Navigating Our Branches**
+
+* If this is a hotfix, make your PR against `master`.
+* If your change can ship immediately, make your PR against `dev`.
```

Comment:
or `dev-tdvt`

Reply:
Changed it. I wonder if we'd like to have `dev-sdk` and `dev-packaging` as well? We could do away with the general `dev` branch in that case, and it'd be clear where things should go/where to branch off of.